src/lib/users/widgets.rb

```diff
@@ -210,5 +196,22 @@ def empty?
     def allow_empty?
       @allow_empty
     end
+
+    # Current password value for root_user
+    #
+    # @return [String]
+    def current_password
+      pwd = root_user.password&.value&.content
```

Comment:
I am just not sure if it will be used also during firstboot? If so, then there is chance that root password is preset to some value and in such case can it be encrypted? Can such case happen?

Reply:
Good question. I don't think so. It not make sense to use _inst\_root\_first_ dialog during firstboot now since It is current purpose is to give the system installed and pre-configured to the final user, not to an administrator. Isn't it? 

It would make sense If firstboot were used to perform the installation, splitting current process in two parts as we already has discussed: to install the very minimum in a first step and then continue with firstboot to fully install and configure the system.

Anyway, I propose to check this in the future, once we have the y2users branch in a better shape.